stories/LinkComponent-stories.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Echoes React
+ * Copyright (C) 2023-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+import { Link, LinkSubdued as LinkSubduedComp } from '@sonarsource/echoes-react';
+import type { Meta, StoryFn, StoryObj } from '@storybook/react';
+import { OpenNewTabIcon } from '../src/common/icons/OpenNewTabIcon';
+
+const meta: Meta<typeof Link> = {
+  component: Link,
+  title: 'Links',
+  args: {
+    to: '/path',
+  },
+} satisfies Meta<typeof Link>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const LinkDefault: Story = {
+  args: {
+    children: 'Link Default',
+    to: '/path',
+  },
+};
+
+export const LinkDefaultWithIcon: Story = {
+  args: {
+    children: 'Link Default with Icon',
+    to: '/path',
+    icon: <OpenNewTabIcon />,
+  },
+};
+
+export const LinkExternal: Story = {
+  args: {
+    children: 'Link External',
+    to: 'https://abc.com/path/new',
+  },
+};
+
+export const LinkSubdued: StoryFn<typeof LinkSubduedComp> = (args) => <LinkSubduedComp {...args} />;
+LinkSubdued.args = {
+  children: 'Link Subdued',
+  to: '/projects/new',
+};
+
+export const LinkSubduedWithIcon: StoryFn<typeof LinkSubduedComp> = (args) => (
+  <LinkSubduedComp {...args} />
+);
+LinkSubduedWithIcon.args = {
+  children: 'Link Subdued With Icon',
+  to: '/projects/new',
+  icon: <OpenNewTabIcon />,
+};
```

Comment:
Since these are different variants I would move them to their own story file and rewrite them to the new story format.

Reply:
Will this create a new component for LikSubdued, should it be part of links ?

Reply:
Yes I think it will create a separate entry in storybook, but that's actually not so bad since it's a different variant that answer different use cases.